KarhooUISDK/Screens/BookingScreen/KarhooBookingViewController.swift

```diff
@@ -362,7 +268,8 @@ public final class KarhooBookingScreenBuilder: BookingScreenBuilder {
             validatedJourneyInfo = journeyInfo
         }
 
-        let bookingPresenter = KarhooBookingPresenter(callback: callback)
+        let router = KarhooBookingRouter()
+        let bookingPresenter = KarhooBookingPresenter(router: router, callback: callback)
```

Comment:
If we want to have test in future it would be nice to be able inject mocked router and presenter. Maybe add them as a constructor parameter with default values?

Reply:
Yeah I was thing about something like this. This way we'd allow DP to provide their own components as well.